gui/packages/desktop/src/renderer/app.js

```diff
@@ -125,10 +114,30 @@ export default class AppRenderer {
     });
 
     ipcRenderer.on('window-shown', () => {
-      this.updateAccountExpiry();
+      if (this._connectedToDaemon) {
```

Comment:
Without that `if` it would attempt to fetch data even before we've connected to daemon.